.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: /usr/src/Slicer-LiverAnalysis
+    docker:
+      - image: olevs/slicer-alive-build
```

Comment:
We should use [slicer/slicer-build](https://hub.docker.com/r/slicer/slicer-build). It has all we need and it is maintained by the Slicer devs

Reply:
slicer/slicer-build don't work, and isn't maintained. It's slicer/slicer-base that is the image to use.
Using this image will add +1,5 hours to each build as the build files are deleted. It don't seem like Slicer keeps any docker images with build files. That's the main reason for creating a new image.

Reply:
To get something up and running I used our fork of Slicer, as the latest changes in the main fork haven't compiled for the last week. I just tried again to compile it inside slicer/slicer-build, and it sill fails.

Reply:
@olevs  you are right, the slicer-build image looks not maintained. Then I suggest the following:

1. Keep all the needed docker image definitions for ALive in a single repository ([ALive-Docker](https://github.com/ALive-research/ALive-Docker)). For now, we can create our own `slicer-build` based on `SlicerDocker/slicer-base`. I would remove  the `slicer-alive-build` in favor of checkout/builld/test commands in the circleci configuration. It is easier to maintain.
2. Have the container `slicer-build` image in [aliveresearch/slicer-build](https://hub.docker.com/repository/docker/aliveresearch/slicer-build) dockerhub repo.

Reply:
I agree. This sounds like a good solution. All the ALive stuff should be lightweight anyway, and don't need to be present in the Docker container.